lib/job_tomate/actions/slack_notify_jira_bug_issue_updated_without_cause.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require "data/user"
+require "support/service_pattern"
+
+module JobTomate
+  module Actions
+
+    # Notifies the JIRA issue's assignee on Slack.
+    class SlackNotifyJIRABugIssueUpdatedWithoutCause
+      extend ServicePattern
+
+      # @param issue [Values::JIRA::Issue]
+      def run(issue)
+        return if issue.assignee_user.nil?
+        if issue.assignee_user.slack_username.blank?
+          LOGGER.warn "unknown Slack username for user ##{issue.assignee_user.id}"
+          return
+        end
+
+        send_message(issue)
+      end
+
+      def send_message(issue)
```

Comment:
I would make `send_message` private to clarify that it's not part of the object's public API.